vpr/src/route/router_lookahead.cpp

```diff
@@ -92,7 +92,7 @@ float MapLookahead::get_expected_cost(int current_node, int target_node, const t
 
     t_rr_type rr_type = device_ctx.rr_nodes[current_node].type();
 
-    if (rr_type == CHANX || rr_type == CHANY) {
+    if (rr_type == CHANX || rr_type == CHANY || rr_type == SOURCE || rr_type == OPIN) {
```

Comment:
Can I suggest that the MapLookahead class definition should move to `router_lookahead_map.h/.cpp`?  It's been on my list to do, and this might be a good time.

Reply:
Yep makes sense!